drivers/hmis/app/graphql/types/hmis_schema/assessment.rb

```diff
@@ -0,0 +1,38 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::Assessment < Types::BaseObject
+    description 'HUD Assessment'
+    field :id, ID, null: false
+    field :client, HmisSchema::Client, null: false
+    field :enrollment, HmisSchema::Enrollment, null: false
+    field :assessment_date, GraphQL::Types::ISO8601DateTime, null: false
+    field :assessment_location, String, null: false
+    field :assessment_type, HmisSchema::Enums::AssessmentType, null: false
+    field :assessment_level, HmisSchema::Enums::AssessmentLevel, null: false
+    field :prioritization_status, HmisSchema::Enums::PrioritizationStatus, null: false
+    field :date_created, GraphQL::Types::ISO8601DateTime, null: false
+    field :date_updated, GraphQL::Types::ISO8601DateTime, null: false
+    field :date_deleted, GraphQL::Types::ISO8601DateTime, null: true
+    # field :user, HmisSchema::User, null: false
+
+    def client
+      load_ar_association(object, :client)
+    end
+
+    def enrollment
+      load_ar_association(object, :enrollment)
+    end
+
+    # TODO: Add user type?
```

Comment:
Yes we're gonig to need this, but I think we can leave it out of this PR. It may need more discussion as to how we handle users in the graphql api (there are both "HUD users" and "HMIS users").